testdata/dnn/llm/gpt2/README.md

```diff
@@ -0,0 +1,24 @@
+# GPT-2 tokenizer 
+
```

Comment:
Also need to add "gpt2_hf_tik_testdata.json" and a section on how to generate this.